spec/Auth.spec.js

```diff
@@ -189,16 +188,56 @@ describe('Auth', () => {
         config: Config.get('test'),
       });
       const roles = [];
-      for(let i = 0; i < rolesNumber;i++){
+      for (let i = 0; i < rolesNumber; i++) {
         const acl = new Parse.ACL();
-        const role = new Parse.Role("roleloadtest" + i, acl);
+        const role = new Parse.Role('roleloadtest' + i, acl);
         role.getUsers().add([user]);
-        roles.push(role.save())
+        roles.push(role.save());
       }
       const savedRoles = await Promise.all(roles);
       expect(savedRoles.length).toBe(rolesNumber);
       const cloudRoles = await userAuth.getRolesForUser();
       expect(cloudRoles.length).toBe(rolesNumber);
     });
+
+    fit('should load all roles for different users with config', async () => {
```

Comment:
Remove `fit`. That makes only this test run and ignore all other

Reply:
Oh shit, I've ran it locally but forget to remove the fit... Sorry

Reply:
Its cool

Reply:
Done